routes/admin.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');// pass passport for configuration
+const router = express.Router();
+const { isLoggedIn, isAdmin } = require('./utils/middleware') ;
+const user = require('./model/usermodel');
+
+
+
+// router.delete('/admin/users',(req,res,next)=>{
+//     user.remove({},(err,user)=>{
+//         res.json(user);
+//     })
+// });
+
+router.get("/admin/addCompany" , (req,res) => {
```

Comment:
These routes must be post and delete routes.
Also, use isLoggedIn and isAdmin middleware on these routes so that they can be accessed by admins only.

Reply:
@SparshKat @thegauravtiwari let's leave the middleware for now. 
@thegauravtiwari I think get routes are required too. Because for example I want to edit a company, then i should first be able to get all companies by /admin/companies and then we'll probably make a edit button in front of each of them which then posts the data. How does that sound?
@SparshKat add the controller functions in adminControllers

Reply:
@VibhorCodecianGupta Sounds good.

Reply:
I think now we need only a get route /admin/companies to get all companies and then we can have edit button on front end to edit the companies and save changes button to make post request on /admin/editCompany . We can change these get requests to post requests.

Reply:
@SparshKat You've already created edit company route. Just change all your admin routes to post routes and add  /admin/companies get route.

Reply:
@thegauravtiwari  won't the edit companies have two set of routes , one get and other the put type of routes? or should i only change them as post routes?

Reply:
so, i will change all of them to put .put routes , do tell me if any other changes are to be done later on ! :)

Reply:
@SparshKat also make the routes snake_case instead of camelCase

Reply:
okay! on it!